Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md

```diff
@@ -703,18 +703,18 @@ TODO add levels & remove automotive-only character
 |:-------------|:-------------|:---------------|:-------------|:------------|
 | **SCORE** | **0.0** | **0.2** | **0.5** | **1.0** |
 | **LSM1**
-| ✦Does your organization gather information about new vulnerability issues? | ??? | ??? | ??? | ??? |
-| ✦Are these issues filtered to dedicated project, used hardware and software stacks? | ??? | ??? | ??? | ??? |
-| ✦Are there project teams after final (serial) release responsible for these issues? | ??? | ??? | ??? | ??? |
+| ✦Does your organization gather information about new vulnerability issues? | NO | SOME | HALF  | MOST |
+| ✦Are these issues filtered to dedicated project, used hardware and software stacks? | NO | SOME | HALF  | MOST|
+| ✦Are there project teams after final (serial) release responsible for these issues? | NO | SOME | HALF  | MOST |
```

Comment:
"NO SOME HALF MOST" is one option to answer. There are more options in the Assessment Excel sheet, as goes:
![image](https://user-images.githubusercontent.com/16705095/58404927-95d08100-8066-11e9-8039-c2d2f40f6236.png)

Reply:
I'll rethink these points